crates/autopilot/src/infra/solvers/dto/settle.rs

```diff
@@ -11,6 +11,10 @@ pub struct Request {
     /// Unique ID of the solution (per driver competition), to settle.
     #[serde_as(as = "serde_with::DisplayFromStr")]
     pub solution_id: u64,
+    /// The maximum number of blocks the autopilot waits for a settlement to
+    /// appear on chain.
+    #[serde_as(as = "serde_with::DisplayFromStr")]
```

Comment:
why is this needed? Do you want to encode this as a string?

Reply:
@fleupold this is a quick draft I pushed just to see how many tests would get affected, it is not the real implementation at all :sweat_smile: